src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -110,7 +110,11 @@ export const BackgroundBlurProvider: FC<React.PropsWithChildren<Props>> = ({
     logger.info(
       `Initializing background blur processor with, spec: ${JSON.stringify(
         spec
```

Comment:
Do you want to break this object out also?

Reply:
I think `spec` is fine, it only contains string and number.